### DISCUSSION TITLE
[Question] Clickable nav item

### DISCUSSION THREAD

**Discussion by ghost** [tabler/tabler#1488] [Q&A]
![11](https://user-images.githubusercontent.com/91934182/220548100-374fa18d-099c-450f-aaca-a0caa6a79a3f.png)

How can I make it so that the dropdown menu opens when I hover the mouse over it?
Like here: https://www.woltlab.com/community/

The other question is (highlighted in red), how can I make the menu item clickable?

We would be fine with the menu opening without a click, so the menu item can be made clickable.

I hope the question is ok here.

Thanks for your work.

LG

**Comment by JobberRT** [comment]
For "click and expand" dropdown, you can go visit [tabler-preview](https://preview.tabler.io/docs/dropdowns.html), locate one of the dropdown you prefer and copy it's html code then adjust it to your need.

For "hover and expand" dropdown, I think it needs write a script function with bootstrap api to manually call `Dropdown.show()` and `Dropdown.hide()` when hover like below:
```javascript
// when using tabler you should do this: globalThis.bootstrap.Dropdown.getXXX
const dd = bootstrap.Dropdown.getOrCreateInstance(YOUR DOM HERE)

// call when hover
dd.show()
// call when leaving
dd.hide()
```
